po/ko.po

```diff
@@ -4001,12 +4023,14 @@ msgid "// <-- the comment makes rustfmt add a newline\n"
 msgstr ""
 
 #: src/tuples-and-arrays/exercise.md:42 src/tuples-and-arrays/solution.md:39
+#, fuzzy
 msgid "\"matrix: {:#?}\""
-msgstr ""
+msgstr "'expr: {:?}'"
```

Comment:
I randomly picked this one, and found it's wrong.

Reply:
Same here.